hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -69,12 +69,13 @@ if {$DEFINE != ""} {
   analyze -sverilog  +define+AST_BYPASS_CLK -f ${SV_FLIST}
 }
 
+# prim_generic_ram_2p is not recognized as a CDC module by the tool
+# So, it's black-boxed while waiting for a tool patch
```

Comment:
do we have an issue to capture and track this?
if yes, a common format that we use is `# TODO(# <issue nr>): Short description`

Reply:
Fixed. Thanks